tox.ini

```diff
@@ -29,10 +29,10 @@ deps =
 
 
 [testenv:docs]
-deps = 
+deps =
 	-r docs/requirements.txt
 commands = 
-	sphinx-build -W -b html -d {envtmpdir}/doctrees docs {envtmpdir}/html
+	sphinx-build -b html -d {envtmpdir}/doctrees docs {envtmpdir}/html
```

Comment:
the `-W` flag is to turn warnings into errors (we don't want that): https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-W

Note that even when I turned this off, there were a couple of real errors hiding underneath (see comment above)